### PR TITLE
Eversong Woods missing SAI

### PR DESCRIPTION
**Pull request by Knindzagxg**
Started checking lower level zones for missing SAI or other problems.

Using Kiera DB to make new, so mods, feel free to edit SQL if need.

To test:
.tele Eversongwoods -> Check creatures around or spawn those with command.

Apply SQL and then restart server (Or reload creature template for every ID)

### CODE REVIEW

**Comment by Pondaveia** [comment]
What changes have you made  by the way? :Smile:

**Comment by Knindzagxg** [comment]
Even here damn :D

It's missing SAI, they just autoattack :D

**Comment by Pondaveia** [comment]
Gotta make tester's job easy ^^

You can simply say:
- <Creature> missing spells
- <Creature> missing spells
- <Creature> missing spells
- <Creature> roll play event

As long as testers know what to do without reading SQL that's fine. But thanks for contribution!

**Comment by Knindzagxg** [comment]
Aye no worries.

Mostly i just made some sql fixes / updates and give to someone to push PTR.
But now people want me to push it as well, so next time i'll give more detailed information :P

**Comment by poszer** [comment]
Hmm, tested in game but there is something wrong for 2 NPC's. 
Before : All of these listed creatures just normally attacking, without using any NPC ability.

Checking the infos at WOTLK DB, and Wiki, i see that just one NPC using ability:

- ENTRY 15650 Crazed Dragonhawk  [Feather Burst](https://wotlk.evowow.com/?npc=15650#abilities) - And it's correctly casted at player with your changes.

But for the next 2 creatures, these SAI isn't correct:

- ENTRY 15668 Grimscale Murloc [NO ABILITIES](https://wotlk.evowow.com/?npc=15668#abilities) "Also, they re casting on ''SELF'' fear when you enter in combat, which is totally wrong how i know.

- ENTRY 15636 Eversong Green Keeper [NO ABILITIES](https://wotlk.evowow.com/?npc=15636#abilities) - You set the ability [Rejuvenation](https://wotlk.evowow.com/?spell=12160). 
That is also incorrect, they can't heal self.

I'll suggest you to remove these 2 scripts (15668 , 15636) and leave just one for the [Crazed Dragonhawk](https://wotlk.evowow.com/?npc=15650) bcoz it's the only correct there.

And thank you for this fix @Knindzagxg

File: data/sql/updates/pending_db_world/rev_1556033703817978000.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 15650;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 15650);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(15650, 0, 0, 0, 0, 0, 100, 0, 3100, 3100, 22800, 22800, 11, 29117, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Crazed Dragonhawk - In Combat - Cast \'29117\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 15668 (name: Grimscale Murloc)
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1556033703817978000.sql]: Remove this one

File: data/sql/updates/pending_db_world/rev_1556033703817978000.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 15668;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 15668);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(15668, 0, 0, 0, 0, 0, 100, 0, 7500, 7500, 45000, 45000, 11, 26661, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Grimscale Murloc - In Combat - Cast \'26661\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 15636 (name: Eversong Green Keeper)
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1556033703817978000.sql]: And this one

**Comment by Knindzagxg** [comment]
Always check wowdb and casts.

Wowhead is good, but wowdb is correct cause u can't miss data there.
NPC that heal him self, for example:

https://www.wowdb.com/npcs/15636-eversong-green-keeper#related:casts

**Comment by poszer** [comment]
> Always check wowdb and casts.
> 
> Wowhead is good, but wowdb is correct cause u can't miss data there.
> NPC that heal him self, for example:
> 
> https://www.wowdb.com/npcs/15636-eversong-green-keeper#related:casts

Hmm, i use this [Evowow db](https://wotlk.evowow.com/?npc=15636) so what's the correct now ? :rofl:

**Comment by Meltie2013** [comment]
Just to point this out, data on wowdb supports the current version running on retail servers. So the data is completely different. Just saying.

**Comment by Stoabrogga** [comment]
That's right: We should be careful to only use data verified for patch 3.3.5a.

**Comment by Meltie2013** [comment]
Well very true in a way, but people forget wowhead is not accurate in information either.
Same goes with WoWDB as well.

- Grimscale Murloc - just has Fear & Flee for Assist (No Repeat)
- Eversong Green Keeper - didn't get Rejuvenation till later patches. Cataclysm or MoP I believe.

Eversong Woods only got updates in Cataclysm with flight paths and some new loot drops.

**Comment by Knindzagxg** [comment]
It was updated with AI with pre-cata patch.
Mostly, spells ID are different if they are added in later expansions.

Later expansions have even more numbers in spell ID, which tells you that got it during Legion or etc.

**Comment by Meltie2013** [comment]
That would be 4.0.1 not 3.3.5a.

**Comment by Knindzagxg** [comment]
It's was up before pre-cata, otherwise we would not have spell in Database.

**Comment by Knindzagxg** [comment]
Updated.

Will keep using wotlk evo from now, just to be a bit more sure.

**Comment by BarbzYHOOL** [comment]
wotlk evo uses trinitycore DB so keep it in mind :P

**Comment by Knindzagxg** [comment]
Which one is best to follow btw?

**Comment by BarbzYHOOL** [comment]
wotlk evowow imo, it's probably up to date with TC's at least

if you need above 3.3.5 info then I usually check wowhead and wowpedia changelogs

**Comment by Stoabrogga** [comment]
Successfully tested, the Crazed Dragonhawk now uses "Feather Burst" in combat.

**Comment by BarbzYHOOL** [comment]
Thank you @Knindzagxg for the PR

thank you @Stoabrogga for testing

**Pull request merged**